src/AstTransforms.cpp

```diff
@@ -226,6 +226,7 @@ bool MaterializeAggregationQueriesTransformer::materializeAggregationQueries(
             if (!needsMaterializedRelation(agg)) {
                 return;
             }
+
```

Comment:
While the added line is reasonable, it's not relevant to your PR. Don't do this in general.